tests/integration/support/navigate-back-to.js

```diff
@@ -0,0 +1,12 @@
+Cypress.Commands.add('navigateBackTo', (resourceUrl, resourceName) => {
+  cy.get('ui5-breadcrumbs')
+    .find(`ui5-link[href*=${resourceUrl}]`, {
+      includeShadowDom: true,
```

Comment:
try to add `includeShadowDom: true` into `cypress.config.js` I made that in ShellBar PR and it works, so probably all of this includes could be potentially removed.